packages/react/src/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,156 @@
+.Tooltip[popover].\:popover-open {
+  /* selector in firefox and safari is \:popover-open */
+  display: block;
+  animation-name: tooltip-appear;
+  animation-duration: 0.1s;
+  animation-fill-mode: forwards;
+  animation-timing-function: ease-in;
+  animation-delay: 0s;
+}
+
+/* This is needed to keep the tooltip open when the user leaves the trigger element to hover tooltip */
+.Tooltip::after {
+  position: absolute;
+  display: block;
+  right: 0;
+  left: 0;
+  height: 8px; /* This is a "magic number". Just gives some room to make sure the tooltip stay open when mouse leaves the trigger to hover tooltip */
+  content: '';
+}
+
+/* Directional styles */
+.Tooltip[data-direction='n'] {
+  border-image: conic-gradient(var(--c) 0 0) fill 0 / var(--r) calc(100% - var(--p) - var(--b) / 2) 0
+    calc(var(--p) - var(--b) / 2) / 0 0 var(--h) 0;
+  clip-path: polygon(
+    0 100%,
+    0 0,
+    100% 0,
+    100% 100%,
+    min(100%, var(--p) + var(--b) / 2) 100%,
+    var(--p) calc(100% + var(--h)),
+    max(0%, var(--p) - var(--b) / 2) 100%
+  );
+}
+
+.Tooltip[data-direction='s'] {
+  border-image: conic-gradient(var(--c) 0 0) fill 0/ 0 calc(100% - var(--p) - var(--b) / 2) var(--r) calc(
+      var(--p) - var(--b) / 2
+    ) / var(--h) 0 0 0;
+  clip-path: polygon(
+    0 0,
+    0 100%,
+    100% 100%,
+    100% 0,
+    min(100%, var(--p) + var(--b) / 2) 0,
+    var(--p) calc(-1 * var(--h)),
+    max(0%, var(--p) - var(--b) / 2) 0
+  );
+}
+
+.Tooltip[data-direction='w'] {
+  border-image: conic-gradient(var(--c) 0 0) fill 0 / calc(var(--p) - var(--b) / 2) 0
+    calc(100% - var(--p) - var(--b) / 2) var(--r) / 0 var(--h) 0 0;
+  clip-path: polygon(
+    100% 100%,
+    0 100%,
+    0 0,
+    100% 0,
+    100% max(0%, var(--p) - var(--b) / 2),
+    calc(100% + var(--h)) var(--p),
+    100% min(100%, var(--p) + var(--b) / 2)
+  );
+}
+
+.Tooltip[data-direction='e'] {
+  border-image: conic-gradient(var(--c) 0 0) fill 0 / calc(var(--p) - var(--b) / 2) var(--r)
+    calc(100% - var(--p) - var(--b) / 2) 0/ 0 0 0 var(--h);
+  clip-path: polygon(
+    0 100%,
+    100% 100%,
+    100% 0,
+    0 0,
+    0 max(0%, var(--p) - var(--b) / 2),
+    calc(-1 * var(--h)) var(--p),
+    0 min(100%, var(--p) + var(--b) / 2)
+  );
+}
+
+.Tooltip[data-direction='n']::after {
+  top: 100%;
+}
+
+.Tooltip[data-direction='s']::after {
+  bottom: 100%;
+}
+
+.Tooltip[data-direction='w']::after {
+  position: absolute;
+  display: block;
+  height: 100%;
+  width: 8px; /* This is a "magic number". Just gives some room to make sure the tooltip stay open when mouse leaves the trigger to hover tooltip */
+  content: '';
+  bottom: 0;
+  left: 100%;
+}
+
+.Tooltip[data-direction='e']::after {
+  position: absolute;
+  display: block;
+  height: 100%;
+  width: 8px;
+  content: '';
+  bottom: 0;
+  right: 100%;
+  margin-left: -8px; /* This is a "magic number". Just gives some room to make sure the tooltip stay open when mouse leaves the trigger to hover tooltip */
```

Comment:
As we're using this value in quite a few places, can it be turned into a locally scoped variable instead which maps to ` var(--base-size-8)`?